luthier/config/src/metrics.lua

```diff
@@ -84,7 +89,10 @@ return {
     NO2M = {
         maker = "doubleSum",
         druidMetric = "NO2"
-    },
+    }
+}
+
+local post_aggregation_metrics = {
```

Comment:
This naming (`aggregation` vs `post-aggregation`) is problematic. It seems to imply that
these metrics are different because of what shows up in their Druid formula, but that's not the
case. The thing that makes these metrics different is whether or not they rely on other Fili
metrics. There is nothing stopping a customer (I don't think) from building a super complex
formula with a whole mess of aggregations and post-aggregations that relies only on Druid metrics.
That would still be an `aggregation` metric in this parlance, even though it has post aggregations
as well. In the original design, and in the comments here I believe, we went with `simple` vs.
`complex` I believe. Maybe that's not a good name either, but at least it doesn't directly
invoke the potential makeup of the Druid query.

Though I don't think having an explicit distinction with its own  field is necessary anyway. Fili doesn't
make the destinction, and the `druidMetrics` field was originally intended simply as an alias for
`dependencies` to keep things clear. I think we need to make a distinction between the two at a
system level, only at a human level.

Reply:
This asymmetry is addressed by merging the two groups. So is the `druidMetric` vs `dependencies` are unified into a `dependencyMetricNames` list as we do before.